2016/01/06 23:48:33 [emerg] 9825#0: bind() to 0.0.0.0:2016 failed (48: Address already in use)
2016/01/06 23:48:33 [emerg] 9825#0: bind() to 0.0.0.0:2016 failed (48: Address already in use)
2016/01/06 23:48:33 [emerg] 9825#0: bind() to 0.0.0.0:2016 failed (48: Address already in use)
2016/01/06 23:49:13 [error] 9984#0: *1 lua entry thread aborted: runtime error: ./src/main.lua:2: attempt to index field 'headers' (a nil value)
stack traceback:
coroutine 0:
	./src/main.lua: in function <./src/main.lua:1>, client: 127.0.0.1, server: , request: "HEAD /main HTTP/1.1", host: "localhost:2016"
